docs/source/2019Barry.rst

```diff
@@ -120,6 +145,28 @@ Hurricane Track and Intensity
     * The time variation of maximum wind speed in GFS.v16.0.10 agrees well with Best Track.
     * The minimum sea level pressure reaches to a lower value in GFS.v16.0.10 (982 hPa) compared with Best Track (993 hPa). 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2019Barry/tracker_Barry_srwv1.png
+      :width: 400
+      :align: center
+
+      Hurricane tracks from SRW_RRFS_v1alpha (blue line), SRW_GFSv15p2 (purple dash line), MRW_GFSv15p2 (red line), and Best Track (black line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+
+    * Both MRW_GFSv15p2 and SRW_GFSv15p2 generate right-of-track biases. 
+    * Hurricane track and intensity simulated by SRW_RRFSv1apha are closer to Best Track compared with SRW_GFSv15p2 and MRW_GFSv15p2.
+
+
+    .. figure:: images/2019Barry/tracker_ws_mslp_Barry_srwv1.png
```

Comment:
I ended up using Best Track data from Jul 11 00Z- Jul 14 18Z. If I remember correctly, there is not available data after Jul 14 18Z. So it is technically 90 hr track data. Could you please cut the lines from SRW runs to be consistent with the available Best Track data?

Reply:
Yes, we can adjust that.